data/assets/scene/digitaluniverse/abell.asset

```diff
@@ -27,18 +27,18 @@ local object = {
     File = speck .. "/abell.speck",
     Texture = textures .. "/point3A.png",
     LabelFile = speck .. "/abell.label",
+    Unit = "Mpc",
     TextColor = { 0.0, 0.8, 0.0 },
     TextSize = 22,
-    TextMinSize = 10.0,
-    Unit = "Mpc",
+    TextMinMaxSize = { 10, 15 },
```

Comment:
OBS! Here I did a stylistic choice to change the max from the previous default (20 px) to 15 pixels, to make the labels not overlap as much. Realizing that this might actually also affect the appearance on really high-res display system, with a risk of being to small. 

Change back to 20?

Reply:
I think it's fine;  15 or 20 don't make too much of a difference..  Was looking at it on a 4K display and the text didn't stand out as too small (didn't even notice the difference, in fact)

Reply:
Not changing back, as the change makes them look better on most displays.